cmd/lintwalk/main.go

```diff
@@ -29,10 +29,9 @@ func getPackagePrefix(dir string) string {
 
 // Main implements gocritic sub-command entry point.
 func Main() {
-	enable := flag.String("enable", "all",
-		`forwarded to linter "as is"`)
-	exclude := flag.String("exclude", "testdata/|vendor/|builtin/",
-		`regexp used to skip package names`)
+	enable := flag.String("enable", "all", `forwarded to linter "as is"`)
+	exclude := flag.String("exclude", "testdata/|vendor/|builtin/", `regexp used to skip package names`)
+	scanGenerated := flag.Bool("scanGenerated", false, ``)
```

Comment:
At least mention that this flag is forwarded to the linter.
We can change API to accept something like `-linterFlags=...` that accepts all flags to be forwarded like Go `-gcflags` do.

Reply:
Do you mean a comment?

Reply:
Flag documentation string is empty right now.
We can avoid flag duplication problem with single arg forwarding mechanism (as in Go tool `gcflags`), so we only need to document one flag, or have several corresponding forwarded flags with short documentation (like `forwarded to linter "as is"`).

Reply:
```diff
- scanGenerated := flag.Bool("scanGenerated", false, ``)
+ scanGenerated := flag.Bool("scanGenerated", false, `forwarded to linter "as is"`)
```